### PR TITLE
Disable Convergence Plots by default

### PR DESCRIPTION
**Pull request by atharva-2001**
Disable progress bars and convergence plots by default. The log level is changed in https://github.com/tardis-sn/tardis/pull/1843. 

**Description**

**Motivation and context**

**How has this been tested?**
- [ ] Testing pipeline.
- [ ] Other. 

**Examples**

**Type of change**

- [ ] Bug fix. 
- [ ] New feature. 
- [ ] Breaking change. 
- [ ] None of the above. 

**Checklist**

- [ ] My change requires a change to the documentation.
    - [ ] I have updated the documentation accordingly.
    - [ ] (optional) I have built the documentation on my fork following [the instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html).
- [ ] I have assigned and requested two reviewers for this pull request.

### CODE REVIEW

File: tardis/base.py [hunk 1]
```diff
@@ -12,10 +12,11 @@ def run_tardis(
     packet_source=None,
     simulation_callbacks=[],
     virtual_packet_logging=False,
-    show_convergence_plots=True,
+    show_convergence_plots=None,
     log_level=None,
     specific_log_level=None,
-    show_progress_bars=True,
+    show_progress_bars=None,
+    tardis_gui=False,
```
Comment (jaladh-singhal) [inline tardis/base.py:19]: You shouldn't really add another option - we have so many already. You can just set default values of both `show_convergence_plots` and `show_progress_bars` to False

Comment (atharva-2001) [reply tardis/base.py]: Done!

File: tardis/base.py [hunk 2]
```diff
@@ -12,10 +12,10 @@ def run_tardis(
     packet_source=None,
     simulation_callbacks=[],
     virtual_packet_logging=False,
-    show_convergence_plots=True,
+    show_convergence_plots=False,
```
Comment (wkerzendorf) [inline tardis/base.py:15]:
```suggestion
    show_convergence_plots=True,
```

File: tardis/base.py [hunk 3]
```diff
@@ -12,10 +12,10 @@ def run_tardis(
...
     log_level=None,
     specific_log_level=None,
-    show_progress_bars=True,
+    show_progress_bars=False,
```
Comment (wkerzendorf) [inline tardis/base.py:18]:
```suggestion
    show_progress_bars=True,
```

File: tardis/base.py [hunk 4]
```diff
@@ -51,9 +51,9 @@ def run_tardis(
         If True, only show the log messages from a particular log level, set by `log_level`.
         If False, the logger shows log messages belonging to the level set and all levels above it in severity.
         The default value None means that the `specific_log_level` specified in the configuration file will be used.
-    show_convergence_plots : bool, default: True, optional
+    show_convergence_plots : bool, default: False, optional
         Option to enable tardis convergence plots.
-    show_progress_bars : bool, default: True, optional
+    show_progress_bars : bool, default: False, optional
```
Comment (wkerzendorf) [inline tardis/base.py:56]:
```suggestion
    show_progress_bars : bool, default: True, optional
```

**Review by andrewfullard — APPROVED** [review]

**Review by jaladh-singhal — APPROVED** [review]

**Pull request merged**